### ISSUE TITLE
Neopixel Control

### ISSUE THREAD

**Issue by MarnitzRoux** [AronHetLam/ATEM_tally_light_with_ESP8266#41]
I'm wanting to use two 8mm PTH Neopixel LEDs for my tally, one facing forward to the speaker and one facing backwards towards the camera operator. I know it would be possible to send the data signal to both LEDs and just have them mimic each other, however, I want to have a lower brightness for the camera operator since the LEDs are quite bright and the operator is much closer than the speaker.

EDIT: I realised there were already implementations for what I was asking

I've looked through the other closed issues and haven't found others that address my questions specifically so I made this one. Let me know if it's been covered before.

TL;DR: Is there a way to control the brightness of the two LEDs independently?

**Comment by AronHetLam** [comment]
Currently no, there's only an overall brightness control for all neopixels at once, but it's possible to implement.
You could hardcode it in your version, that would be the simplest for now.
It sounds like you ably want two neopixels, and none of them to act as a status led. I'll post a line of code and where to put it later today if you need it 🙂

**Comment by MarnitzRoux** [comment]
Yeah, similar to the functionality of the regular LEDs where you can choose the modes, where you can choose to show program, preview, or both.
The argument can be made that if the LED facing the speaker is only showing program, I can just use a regular red LED but I want to keep that functionality for future use-cases. And because the only 8mm LEDs I have are Neopixels.

I think I've figure it out though, I can modify the `setSTRIP` function to `setNEO1` to only apply to the first Neopixel and make a second `setNEO2` function to control the second Neopixel and set them for different LED modes so they can function differently on the same data pin. I'll only be able to test that on Friday though.

**Comment by AronHetLam** [comment]
Cool. Something like that would be my quick fix as well.
